skimage/feature/censure.py

```diff
@@ -153,7 +153,7 @@ class CENSURE(FeatureDetector):
     .. [2] Adam Schmidt, Marek Kraft, Michal Fularz and Zuzanna Domagala
            "Comparative Assessment of Point Feature Detectors and
            Descriptors in the Context of Robot Navigation"
-           http://www.jamris.org/01_2013/saveas.php?QUEST=JAMRIS_No01_2013_P_11-20.pdf
+           http://yadda.icm.edu.pl/yadda/element/bwmeta1.element.baztech-268aaf28-0faf-4872-a4df-7e2e61cb364c/c/Schmidt_comparative.pdf
```

Comment:
For №1: I'd change the url to https://link.springer.com/chapter/10.1007/978-3-540-88693-8_8 and add `DOI:10.1007/978-3-540-88693-8_8` afterwards.

For №2: I'd keep the current link and add `DOI:10.1.1.465.1117`.

@stefanv I assume @SeraYang1 is speaking about link to .pdf, not to the page.